main.rb

```diff
@@ -0,0 +1,159 @@
+# rubocop:disable Metrics\CyclomaticComplexity, Metrics/MethodLength
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    puts "Welcome to School Library App!\n\n"
+    command = nil
+    while command != '7'
+      commands
+
+      command = gets.chomp
+      case command
+      when '1'
+        list_books
+      when '2'
+        list_people
+      when '3'
+        create_person
+      when '4'
+        create_book
+      when '5'
+        create_rental
+      when '6'
+        list_rentals
+      when '7'
+        puts 'Thank you for using this app!'
+      else
+        puts 'Invalid option'
+      end
+    end
+  end
+
+  def commands
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
```

Comment:
- [ ] I am not able to create a teacher as well when it comes to adding the specialization that app crashes, kindly let's fix this issue 👍